test/cell_test.rb

```diff
@@ -56,20 +56,20 @@ def test_fired_upon_starts_false
   end
 
   def test_fire_upon_lowers_ship_health_by_one
-    @cell.place_ship(@submarine)
+    @cell.place_ship(@cruiser)
```

Comment:
I think we can go back to @submarine here, so that the following assertion still works; if we switch to cruiser, it will go down to 2 instead of the expected 1